src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala

```diff
@@ -315,20 +315,10 @@ trait BasePage extends Matchers with BrowserDriver {
     signInButton.click()
   }
 
-  def createUser(): Unit = {
-    clickByCSS("#affinityGroup-Individual")
-    clickById("principalEnrolments[0].key-HMRC-CTS-ORG")
-    clickByCSS("#update")
-    enrollment.clear()
-    enrollment.sendKeys("GB123456789012345")
-  }
-
```

Comment:
What is happening to setting the user enrolment? Is this method not used anywhere and there is a different method that covers this logic?

Reply:
Once I removed it from the login step, it was not used anywhere else